### PR TITLE
chore: fix: test(er/ut): improve coverage of resource codes

### PR DESCRIPTION
**Pull request by Lance52259**
**What this PR does / why we need it**:
Some resource logics is incorrect and the coding coverage of related acceptance tests are less than 70%.

Logics update:
1. Resource of the static route is missing refresh function to wait the status from 'pending' to 'available'
2. Time result needs to format to remove the millseconds.
3. Before instance and the custom route table creates complete, we cannot update the default route tables. So, the logic of update default route table after instance create complete will never be used.

**Which issue this PR fixes**:
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
fixes #xxx

**Special notes for your reviewer**:

**Release note**:

```release-note
1. improve coverage of resource codes.
```

## PR Checklist

* [x] Tests added/passed.
* [x] Documentation updated.
* [x] Schema updated.

## Acceptance Steps Performed

```
./coverage_test.sh 
>>> Start to test
=== RUN   TestAccDataSourceAssociations_basic
=== PAUSE TestAccDataSourceAssociations_basic
=== RUN   TestAccDataSourceAttachments_basic
=== PAUSE TestAccDataSourceAttachments_basic
=== RUN   TestAccDataSourceAZs_basic
=== PAUSE TestAccDataSourceAZs_basic
=== RUN   TestAccDataSourceAvailableRoutes_basic
=== PAUSE TestAccDataSourceAvailableRoutes_basic
=== RUN   TestAccDatasourceFlowLogs_basic
=== PAUSE TestAccDatasourceFlowLogs_basic
=== RUN   TestAccDataSourceInstances_basic
=== PAUSE TestAccDataSourceInstances_basic
=== RUN   TestAccDataSourcePropagations_basic
=== PAUSE TestAccDataSourcePropagations_basic
=== RUN   TestAccDataSourceQuotas_basic
=== PAUSE TestAccDataSourceQuotas_basic
=== RUN   TestAccDataSourceRouteTables_basic
=== PAUSE TestAccDataSourceRouteTables_basic
=== RUN   TestAccAssociation_basic
=== PAUSE TestAccAssociation_basic
=== RUN   TestAccFlowLog_basic
=== PAUSE TestAccFlowLog_basic
=== RUN   TestAccInstance_basic
=== PAUSE TestAccInstance_basic
=== RUN   TestAccInstance_updateWithEpsId
=== PAUSE TestAccInstance_updateWithEpsId
=== RUN   TestAccPropagation_basic
=== PAUSE TestAccPropagation_basic
=== RUN   TestAccRouteTable_basic
=== PAUSE TestAccRouteTable_basic
=== RUN   TestAccStaticRoute_basic
=== PAUSE TestAccStaticRoute_basic
=== RUN   TestAccVpcAttachment_basic
=== PAUSE TestAccVpcAttachment_basic
=== CONT  TestAccDataSourceAssociations_basic
=== CONT  TestAccAssociation_basic
--- PASS: TestAccAssociation_basic (181.74s)
=== CONT  TestAccDataSourceInstances_basic
--- PASS: TestAccDataSourceAssociations_basic (261.32s)
=== CONT  TestAccDataSourceRouteTables_basic
--- PASS: TestAccDataSourceInstances_basic (107.23s)
=== CONT  TestAccDataSourceQuotas_basic
--- PASS: TestAccDataSourceRouteTables_basic (221.43s)
=== CONT  TestAccDataSourceAvailableRoutes_basic
--- PASS: TestAccDataSourceQuotas_basic (215.06s)
=== CONT  TestAccDatasourceFlowLogs_basic
--- PASS: TestAccDataSourceAvailableRoutes_basic (206.01s)
=== CONT  TestAccDataSourceAZs_basic
--- PASS: TestAccDataSourceAZs_basic (12.39s)
=== CONT  TestAccDataSourceAttachments_basic
--- PASS: TestAccDatasourceFlowLogs_basic (235.74s)
=== CONT  TestAccPropagation_basic
--- PASS: TestAccDataSourceAttachments_basic (210.43s)
=== CONT  TestAccVpcAttachment_basic
--- PASS: TestAccPropagation_basic (187.21s)
=== CONT  TestAccStaticRoute_basic
--- PASS: TestAccVpcAttachment_basic (221.11s)
=== CONT  TestAccRouteTable_basic
--- PASS: TestAccStaticRoute_basic (305.25s)
=== CONT  TestAccInstance_basic
--- PASS: TestAccInstance_basic (118.10s)
=== CONT  TestAccInstance_updateWithEpsId
--- PASS: TestAccRouteTable_basic (236.33s)
=== CONT  TestAccFlowLog_basic
--- PASS: TestAccInstance_updateWithEpsId (88.31s)
=== CONT  TestAccDataSourcePropagations_basic
--- PASS: TestAccFlowLog_basic (277.15s)
--- PASS: TestAccDataSourcePropagations_basic (266.30s)
PASS
coverage: 81.2% of statements in ../../er
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/er        1704.998s
github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/er/data_source_huaweicloud_er_associations.go:18:            DataSourceAssociations                          100.0%
github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/er/data_source_huaweicloud_er_associations.go:56:            associationsSchema                              100.0%
github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/er/data_source_huaweicloud_er_associations.go:99:            buildAssociationsistOpts                        100.0%
github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/er/data_source_huaweicloud_er_associations.go:116:           flattenAssociations                             100.0%
github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/er/data_source_huaweicloud_er_associations.go:139:           dataSourceAssociationsRead                      86.7%
github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/er/data_source_huaweicloud_er_attachments.go:19:             DataSourceAttachments                           100.0%
github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/er/data_source_huaweicloud_er_attachments.go:136:            filterAttachments                               92.9%
github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/er/data_source_huaweicloud_er_attachments.go:159:            filterAttachmentsByTags                         100.0%
github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/er/data_source_huaweicloud_er_attachments.go:176:            flattenAttachments                              83.3%
github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/er/data_source_huaweicloud_er_attachments.go:201:            buildAttachmentListOpts                         100.0%
github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/er/data_source_huaweicloud_er_attachments.go:210:            dataSourceAttachmentsRead                       71.4%
github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/er/data_source_huaweicloud_er_availability_zones.go:19:      DataSourceAvailabilityZones                     100.0%
github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/er/data_source_huaweicloud_er_availability_zones.go:39:      dataSourceAvailabilityZonesRead                 81.0%
github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/er/data_source_huaweicloud_er_availability_zones.go:86:      flattenListAvailabilityZone                     87.5%
github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/er/data_source_huaweicloud_er_available_routes.go:20:        DataSourceErAvailableRoutes                     100.0%
github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/er/data_source_huaweicloud_er_available_routes.go:108:       newAvailableRoutesDSWrapper                     100.0%
github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/er/data_source_huaweicloud_er_available_routes.go:115:       dataSourceErAvailableRoutesRead                 83.3%
github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/er/data_source_huaweicloud_er_available_routes.go:137:       ListEffectiveRoutes                             87.5%
github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/er/data_source_huaweicloud_er_available_routes.go:159:       listEffectiveRoutesToSchema                     100.0%
github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/er/data_source_huaweicloud_er_flow_logs.go:20:               DataSourceFlowLogs                              100.0%
github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/er/data_source_huaweicloud_er_flow_logs.go:84:               flowLogSchema                                   100.0%
github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/er/data_source_huaweicloud_er_flow_logs.go:152:              dataSourceFlowLogsRead                          83.3%
github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/er/data_source_huaweicloud_er_flow_logs.go:199:              flattenListTransitIpsResponseBody               87.5%
github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/er/data_source_huaweicloud_er_flow_logs.go:229:              filterListFlowLogsResponseBody                  62.5%
github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/er/data_source_huaweicloud_er_flow_logs.go:263:              buildListFlowLogsQueryParams                    100.0%
github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/er/data_source_huaweicloud_er_instances.go:19:               DataSourceInstances                             100.0%
github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/er/data_source_huaweicloud_er_instances.go:152:              filterInstances                                 93.3%
github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/er/data_source_huaweicloud_er_instances.go:177:              flattenInstances                                83.3%
github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/er/data_source_huaweicloud_er_instances.go:206:              buildSliceIgnoreEmptyElement                    100.0%
github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/er/data_source_huaweicloud_er_instances.go:213:              buildInstanceListOpts                           100.0%
github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/er/data_source_huaweicloud_er_instances.go:223:              dataSourceInstancesRead                         72.2%
github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/er/data_source_huaweicloud_er_propagations.go:18:            DataSourcePropagations                          100.0%
github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/er/data_source_huaweicloud_er_propagations.go:56:            propagationSchema                               100.0%
github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/er/data_source_huaweicloud_er_propagations.go:103:           buildPropagationListOpts                        100.0%
github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/er/data_source_huaweicloud_er_propagations.go:120:           flattenPropagations                             100.0%
github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/er/data_source_huaweicloud_er_propagations.go:144:           dataSourcePropagationsRead                      82.4%
github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/er/data_source_huaweicloud_er_quotas.go:19:                  DataSourceErQuotas                              100.0%
github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/er/data_source_huaweicloud_er_quotas.go:83:                  newQuotasDSWrapper                              100.0%
github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/er/data_source_huaweicloud_er_quotas.go:90:                  dataSourceErQuotasRead                          75.0%
github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/er/data_source_huaweicloud_er_quotas.go:112:                 ShowQuotas                                      85.7%
github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/er/data_source_huaweicloud_er_quotas.go:133:                 showQuotasToSchema                              100.0%
github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/er/data_source_huaweicloud_er_route_tables.go:29:            DataSourceRouteTables                           100.0%
github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/er/data_source_huaweicloud_er_route_tables.go:185:           routeTableRelationshipSchemaResource            100.0%
github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/er/data_source_huaweicloud_er_route_tables.go:207:           queryRouteTableAssociations                     77.8%
github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/er/data_source_huaweicloud_er_route_tables.go:228:           queryRouteTablePropagations                     77.8%
github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/er/data_source_huaweicloud_er_route_tables.go:249:           queryRouteTableRoutes                           76.5%
github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/er/data_source_huaweicloud_er_route_tables.go:286:           filterRouteTablesByTags                         84.6%
github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/er/data_source_huaweicloud_er_route_tables.go:310:           flattenRouteTables                              90.0%
github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/er/data_source_huaweicloud_er_route_tables.go:343:           dataSourceRouteTablesRead                       75.0%
github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/er/resource_huaweicloud_er_association.go:26:                ResourceAssociation                             100.0%
github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/er/resource_huaweicloud_er_association.go:91:                resourceAssociationCreate                       78.6%
github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/er/resource_huaweicloud_er_association.go:131:               QueryAssociationById                            85.7%
github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/er/resource_huaweicloud_er_association.go:163:               associationStatusRefreshFunc                    72.7%
github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/er/resource_huaweicloud_er_association.go:186:               resourceAssociationRead                         76.9%
github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/er/resource_huaweicloud_er_association.go:222:               resourceAssociationDelete                       78.6%
github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/er/resource_huaweicloud_er_association.go:261:               resourceAssociationImportState                  83.3%
github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/er/resource_huaweicloud_er_flow_log.go:29:                   ResourceFlowLog                                 100.0%
github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/er/resource_huaweicloud_er_flow_log.go:112:                  buildCreateFlowLogBodyParams                    100.0%
github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/er/resource_huaweicloud_er_flow_log.go:127:                  resourceFlowLogCreate                           78.1%
github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/er/resource_huaweicloud_er_flow_log.go:184:                  getFlowLogInfo                                  92.9%
github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/er/resource_huaweicloud_er_flow_log.go:210:                  resourceFlowLogRead                             80.0%
github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/er/resource_huaweicloud_er_flow_log.go:245:                  updateFlowLogState                              88.9%
github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/er/resource_huaweicloud_er_flow_log.go:265:                  buildUpdateFlowLogBodyParams                    100.0%
github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/er/resource_huaweicloud_er_flow_log.go:273:                  resourceFlowLogUpdate                           79.3%
github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/er/resource_huaweicloud_er_flow_log.go:325:                  resourceFlowLogDelete                           83.3%
github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/er/resource_huaweicloud_er_flow_log.go:365:                  flowLogStatusRefreshFunc                        72.2%
github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/er/resource_huaweicloud_er_flow_log.go:400:                  flowLogWaitingForStateCompleted                 100.0%
github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/er/resource_huaweicloud_er_flow_log.go:413:                  resourceFlowLogImportState                      80.0%
github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/er/resource_huaweicloud_er_instance.go:36:                   ResourceInstance                                100.0%
github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/er/resource_huaweicloud_er_instance.go:145:                  resourceInstanceCreate                          76.0%
github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/er/resource_huaweicloud_er_instance.go:194:                  buildCreateInstanceBodyParams                   100.0%
github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/er/resource_huaweicloud_er_instance.go:201:                  buildCreateInstanceInstanceChildBody            100.0%
github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/er/resource_huaweicloud_er_instance.go:215:                  instanceWaitingForStateCompleted                81.8%
github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/er/resource_huaweicloud_er_instance.go:263:                  resourceInstanceRead                            81.2%
github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/er/resource_huaweicloud_er_instance.go:317:                  resourceInstanceUpdate                          82.9%
github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/er/resource_huaweicloud_er_instance.go:412:                  buildUpdateInstanceBodyParams                   100.0%
github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/er/resource_huaweicloud_er_instance.go:419:                  buildUpdateInstanceInstanceChildBody            66.7%
github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/er/resource_huaweicloud_er_instance.go:436:                  buildUpdateInstanceAvailabilityZonesBodyParams  100.0%
github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/er/resource_huaweicloud_er_instance.go:443:                  resourceInstanceDelete                          80.0%
github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/er/resource_huaweicloud_er_instance.go:476:                  deleteInstanceWaitingForStateCompleted          81.8%
github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/er/resource_huaweicloud_er_propagation.go:26:                ResourcePropagation                             100.0%
github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/er/resource_huaweicloud_er_propagation.go:91:                resourcePropagationCreate                       78.6%
github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/er/resource_huaweicloud_er_propagation.go:131:               QueryPropagationById                            85.7%
github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/er/resource_huaweicloud_er_propagation.go:163:               propagationStatusRefreshFunc                    72.7%
github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/er/resource_huaweicloud_er_propagation.go:186:               resourcePropagationRead                         76.9%
github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/er/resource_huaweicloud_er_propagation.go:222:               resourcePropagationDelete                       78.6%
github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/er/resource_huaweicloud_er_propagation.go:261:               resourcePropagationImportState                  83.3%
github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/er/resource_huaweicloud_er_route_table.go:35:                ResourceRouteTable                              100.0%
github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/er/resource_huaweicloud_er_route_table.go:117:               buildRouteTableCreateOpts                       100.0%
github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/er/resource_huaweicloud_er_route_table.go:125:               resourceRouteTableCreate                        80.0%
github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/er/resource_huaweicloud_er_route_table.go:156:               routeTableStatusRefreshFunc                     75.0%
github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/er/resource_huaweicloud_er_route_table.go:179:               resourceRouteTableRead                          78.6%
github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/er/resource_huaweicloud_er_route_table.go:213:               updateRouteTableBasicInfo                       87.5%
github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/er/resource_huaweicloud_er_route_table.go:241:               resourceRouteTableUpdate                        76.9%
github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/er/resource_huaweicloud_er_route_table.go:265:               releaseRouteTableAssociations                   44.4%
github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/er/resource_huaweicloud_er_route_table.go:283:               releaseRouteTablePropagations                   44.4%
github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/er/resource_huaweicloud_er_route_table.go:301:               resourceRouteTableDelete                        76.2%
github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/er/resource_huaweicloud_er_route_table.go:342:               resourceRouteTableImportState                   80.0%
github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/er/resource_huaweicloud_er_static_route.go:26:               ResourceStaticRoute                             100.0%
github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/er/resource_huaweicloud_er_static_route.go:98:               buildStaticRouteCreateOpts                      100.0%
github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/er/resource_huaweicloud_er_static_route.go:106:              staticRouteStatusRefreshFunc                    88.9%
github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/er/resource_huaweicloud_er_static_route.go:124:              resourceStaticRouteCreate                       78.6%
github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/er/resource_huaweicloud_er_static_route.go:156:              resourceStaticRouteRead                         81.2%
github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/er/resource_huaweicloud_er_static_route.go:200:              resourceStaticRouteUpdate                       78.6%
github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/er/resource_huaweicloud_er_static_route.go:236:              resourceStaticRouteDelete                       78.6%
github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/er/resource_huaweicloud_er_static_route.go:267:              resourceStaticRouteImportState                  71.4%
github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/er/resource_huaweicloud_er_vpc_attachment.go:29:             ResourceVpcAttachment                           100.0%
github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/er/resource_huaweicloud_er_vpc_attachment.go:120:            resourceVpcAttachmentCreate                     80.0%
github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/er/resource_huaweicloud_er_vpc_attachment.go:157:            resourceVpcAttachmentRead                       72.7%
github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/er/resource_huaweicloud_er_vpc_attachment.go:195:            updateVpcAttachmentBasicInfo                    87.5%
github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/er/resource_huaweicloud_er_vpc_attachment.go:223:            vpcAttachmentStatusRefreshFunc                  83.3%
github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/er/resource_huaweicloud_er_vpc_attachment.go:246:            resourceVpcAttachmentUpdate                     75.0%
github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/er/resource_huaweicloud_er_vpc_attachment.go:269:            resourceVpcAttachmentDelete                     78.6%
github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/er/resource_huaweicloud_er_vpc_attachment.go:299:            resourceVpcAttachmentImportState                80.0%
total:                                                                                                                                  (statements)                                    81.2%
```

### CODE REVIEW

File: huaweicloud/services/er/resource_huaweicloud_er_static_route.go [hunk 1]
```diff
@@ -178,10 +218,22 @@ func resourceStaticRouteUpdate(ctx context.Context, d *schema.ResourceData, meta
 		return diag.Errorf("error updating static route (%s): %s", staticRouteId, err)
 	}
 
+	stateConf := &resource.StateChangeConf{
+		Pending:      []string{"PENDING"},
+		Target:       []string{"COMPLETED"},
+		Refresh:      staticRouteStatusRefreshFunc(client, routeTableId, d.Id(), []string{"available"}),
+		Timeout:      d.Timeout(schema.TimeoutCreate),
```
Comment (deer-hang) [inline huaweicloud/services/er/resource_huaweicloud_er_static_route.go]:
```suggestion
		Timeout:      d.Timeout(schema.TimeoutUpdate),
```

File: huaweicloud/services/er/resource_huaweicloud_er_static_route.go [hunk 2]
```diff
@@ -198,6 +250,17 @@ func resourceStaticRouteDelete(_ context.Context, d *schema.ResourceData, meta i
 		return diag.Errorf("error deleting static route (%s): %s", staticRouteId, err)
 	}
 
+	stateConf := &resource.StateChangeConf{
+		Pending:      []string{"PENDING"},
+		Target:       []string{"COMPLETED"},
+		Refresh:      staticRouteStatusRefreshFunc(client, routeTableId, d.Id(), nil),
+		Timeout:      d.Timeout(schema.TimeoutCreate),
```
Comment (deer-hang) [inline huaweicloud/services/er/resource_huaweicloud_er_static_route.go]:
```suggestion
		Timeout:      d.Timeout(schema.TimeoutDelete),
```

File: huaweicloud/services/er/resource_huaweicloud_er_static_route.go (hunk 1 above)
Comment (Lance52259) [reply huaweicloud/services/er/resource_huaweicloud_er_static_route.go]: done.

File: huaweicloud/services/er/resource_huaweicloud_er_static_route.go (hunk 2 above)
Comment (Lance52259) [reply huaweicloud/services/er/resource_huaweicloud_er_static_route.go]: done.

**Pull request merged**